docs/docs/userguide/INSTALLATION.md

```diff
@@ -11,7 +11,7 @@ Set up a user with an administrator IAM role. See [Configuration basics — AWS
 Clone the `data-center-terraform` project repository from GitHub:
 
 ```shell
-git clone -b 2.0.6 https://github.com/atlassian-labs/data-center-terraform.git && cd data-center-terraform
+git clone -b 2.7.0 https://github.com/atlassian-labs/data-center-terraform.git && cd data-center-terraform
```

Comment:
```suggestion
git clone -b 2.0.7 https://github.com/atlassian-labs/data-center-terraform.git && cd data-center-terraform
```

Reply:
I fixed this - the new version is `2.1.0`